packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -1809,6 +1809,8 @@ test('reinstate vault', async t => {
 
   await aethVaultMetrics.assertChange({
     numActiveVaults: 1,
+    liquidatingDebt: { value: 0n },
```

Comment:
ah, I see now why the `TotalDebtLiquidated` rename was proposed and should be abandoned